charts/kueue/Chart.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: kueue
+description: A Helm chart for Kubernetes
```

as seen at https://helm.sh/docs/topics/charts/#the-chartyaml-file  is the description of the HELM managed project
```suggestion
description: |
Kueue is a set of APIs and controller for job queueing. It is a job-level manager that decides when a job should be admitted to start (as in pods can be created) and when it should stop (as in active pods should be deleted).
```

---

Thanks! fixed it